pkg/sass-parser/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 0.2.6
 
-* No user-visible changes.
+* Add `BooleanExpression` and `NumberExpression`.
```

Comment:
I can update this to 0.2.7 if this version has already come out

Reply:
Yeah, I just cut a release for it.